component-classes/index.js

```diff
@@ -20,15 +20,17 @@ export const ribbon = {
 };
 
 export const badge = {
-  base: 'py-4 px-8 border-0 rounded-4 text-xs inline-flex',
-  neutral: 'i-bg-$color-badge-neutral-background i-text-$color-badge-neutral-text',
-  info: 'i-bg-$color-badge-info-background i-text-$color-badge-info-text',
-  positive: 'i-bg-$color-badge-positive-background i-text-$color-badge-positive-text',
-  warning: 'i-bg-$color-badge-warning-background i-text-$color-badge-warning-text',
-  negative: 'i-bg-$color-badge-negative-background i-text-$color-badge-negative-text',
-  disabled: 'i-bg-$color-badge-disabled-background i-text-$color-badge-disabled-text',
-  price: 'i-bg-$color-badge-price-background i-text-$color-badge-price-text',
-  notification: 'i-bg-$color-badge-notification-background i-text-$color-badge-notification-text',
+  base: 'py-4 px-8 border-0 rounded-4 text-xs inline-flex s-text',
```

Comment:
Have you tested that Badge of price variant gets the right text color? It would probably have both `s-text` and `s-text-inverted` applied, which I guess could cause trouble :D

Reply:
I wanted to avoid duplicating `s-text` or setting `s-text-disabled` to important. And I expect the rules to be sorted alphabetically (within each rule group), isn't that the way it works? 😅

Reply:
I'll revert this to not inflict any side-effects, and we can rework this for v2. 👍

Reply:
> I wanted to avoid duplicating s-text or setting s-text-disabled to important. And I expect the rules to be sorted alphabetically (within each rule group), isn't that the way it works?

I think we've discussed at some point that because of the same specificity of utility classes, we should generally avoid setting classes controlling the same CSS property on one element. That's why button classes look the way they look :D Assuming that the Warp unocss rules order will never change is pretty risky :D